app/views/restaurants/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul>
+  <% @restaurants.each do |restaurant| %>
+    <li><%= link_to "#{restaurant.name}", restaurant_path(restaurant) %></li>
+  <% end %>
+</ul>
+
+<h3><center><%= link_to "New", new_restaurant_path %></center></h3>
```

Comment:
Don't use the html element to center here! Instead, if you want to center this text, add the class `text-center` directly to the `h3`. (This class is provided by Foundation.) It's better form to keep non-content-based stuff out of your html tags